### DISCUSSION TITLE
希望能将漫画内吐槽页面改至1.2.2版本

### DISCUSSION THREAD

**Discussion by FightimeX** [xiaoyaocz/flutter_dmzj#119] [General]
感觉新版吐槽页面不如老版清晰明了
![Screenshot_2023-03-16-11-53-24-324_com xycz dmzjx](https://user-images.githubusercontent.com/128014287/225510331-a7191653-617e-477f-a087-95069b19666a.jpg)
![Screenshot_2023-03-16-11-47-42-147_com nsapp dmzj](https://user-images.githubusercontent.com/128014287/225510344-478f5778-1cc8-4fb4-a436-444bcf660df5.jpg)

**Comment by xiaoyaocz** [comment]
https://github.com/xiaoyaocz/flutter_dmzj/releases/tag/v2.0.3-beta
